scripts/tests/testIntegration.sh

```diff
@@ -8,4 +8,4 @@ json=$(node -e "console.log(require('config').get('firestore'))")
 project_id=$(echo $json | grep -o '"project_id":[^,}]*' | cut -d':' -f2 | tr -d '"' | tr -d '[:space:]')
 
 echo 'Start firestore emulator and run integration tests:'
-firebase emulators:exec 'nyc mocha test/integration/**' --project=$project_id
+firebase emulators:exec 'nyc mocha test/integration/**' --project=$project_id
```

Comment:
I dont see any change here.

Reply:
was it done by linters i.e. last line removed,
But i see in the next file a blank line was added.

Reply:
that was just formatting doesnt seem to be a blocker